extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-aks.ipynb

```diff
@@ -285,10 +297,15 @@
                 "os.environ[\"ACCEPT_EULA\"] = 'yes'\n",
                 "os.environ[\"USERNAME\"] = mssql_username\n",
                 "os.environ[\"PASSWORD\"] = mssql_password\n",
+                "if os.name == 'nt':\n",
+                "    print(f'If you don\\'t see output produced by azdata, you can run the following command in terminal to check the deployment status: kubectl get pods -n {mssql_cluster_name} ')\n",
```

Comment:
>following [](start = 91, length = 9)

"run the following command in the terminal" or "run the following command in a terminal window"

I'd also consider formatting the command in some way so it's clear what the whole command is (even just having it on another line would be helpful)